full_bubble_model.py

```diff
@@ -85,32 +188,29 @@ def Work(cpar):
     V_E = 4.0 / 3.0 * cpar.R_E**3 * np.pi    # [m^3]
     V_0 = 4.0 / 3.0 * R_0**3 * np.pi  # [m^3]
     
-    p_E = cpar.P_inf + 2 * cpar.surfactant * par.sigma / cpar.R_E # [Pa]
-    p_gas = p_E - cpar.P_v # [Pa]
+    p_E = cpar.P_amb + 2 * cpar.surfactant * par.sigma / cpar.R_E # [Pa]
+    p_gas = p_E - cpar.P_v if evaporation else p_E # [Pa]
     n_gas = p_gas * V_E / (par.R_g * cpar.T_inf) # [mol]
     
-    W_gas0 = -(cpar.P_v * V_0 + n_gas * par.R_g * cpar.T_inf * np.log(V_0))
-    W_gasE = -(cpar.P_v * V_E + n_gas * par.R_g * cpar.T_inf * np.log(V_E))
+    W_gas0 = -(cpar.P_v * V_0 + n_gas * par.R_g * cpar.T_inf * np.log(V_0)) if evaporation else -(n_gas * par.R_g * cpar.T_inf * np.log(V_0))
+    W_gasE = -(cpar.P_v * V_E + n_gas * par.R_g * cpar.T_inf * np.log(V_E)) if evaporation else -(n_gas * par.R_g * cpar.T_inf * np.log(V_E))
     W_gas = W_gas0 - W_gasE # [J]
     
     W_surface_tension = par.sigma * cpar.surfactant *  4.0 * np.pi * (R_0**2 - cpar.R_E**2) # [J]
-    W_flow = cpar.P_inf * 4.0 / 3.0 * np.pi * (R_0**3 - cpar.R_E**3) # [J]
+    W_flow = cpar.P_amb * 4.0 / 3.0 * np.pi * (R_0**3.0 - cpar.R_E**3.0) # [J]
     return W_gas + W_surface_tension + W_flow    # [J]
 
 
 """________________________________Pressures________________________________"""
 
-@njit(float64[:](float64, float64, float64, float64, float64, float64, float64, float64, float64))
-def Pressure(R, R_dot, T, T_dot, M, sum_omega_dot, P_inf, mu_L, surfactant):
-    p_inf = P_inf
-    p_inf_dot = 0.0
-    p = 0.1 * M * par.R_erg * T
-    p_dot = p * (sum_omega_dot / M + T_dot / T - 3.0 * R_dot/R)
+@njit(float64[:](float64, float64, float64, float64, float64, float64, float64, float64, float64[:]))
+def Pressure(t, R, R_dot, mu_L, surfactant, p, p_dot, P_amb, args):
```

Comment:
This function is modified to incorporate any pressure excitation defined above in _____Settings_____